src/v2/Apps/Artists/artistsRoutes.tsx

```diff
@@ -69,6 +71,19 @@ export const artistsRoutes: RouteConfig[] = [
             }
           }
         `,
+        render: ({ Component, props, match }) => {
+          if (!(Component && props)) {
+            // Returning `null` will show the spinner; but `undefined` uses the loading bar
```

Comment:
Copypasta; but I thought this odd.

Reply:
Hmm, not sure. Is there some loading state/fallback in the app shell or something?